package.json

```diff
@@ -12,7 +13,10 @@
     "src"
   ],
   "scripts": {
-    "test": "qunit tests/*-test.js",
+    "test": "qunit tests/*-test.js && codemod-cli test",
```

Comment:
maybe we can use something like `npm-run-all` which is used by Ember

Reply:
Yeah that will work too.